iroh-net/src/defaults.rs

```diff
@@ -125,3 +125,77 @@ pub mod staging {
         }
     }
 }
+
+/// Contains all timeouts that we use in `iroh-net`.
+pub mod timeouts {
```

Comment:
`pub(crate)` I think?

I've come to the conclusion it's a lot easier to always be explicit about the visibility of things rather than rely on things like "this is pub but it's inside a pub(crate)" or similar.